package.json

```diff
@@ -47,6 +47,7 @@
     "http-proxy-middleware": "^2.0.6",
     "lodash": "^4.17.21",
     "mapbox-gl": "3.4.0",
+    "mapbox-gl-draw-rectangle-mode": "^1.0.4",
```

Comment:
I see you use the DrawRectangle.js, do you still need to import this or you just import this and delete the DrawRectangle.js?

Reply:
I think we can keep both. The reason why i did this is, importing it in `package.json` is to make it available to dependabot. Leveraging dependabot is easier for us to know whether this `mapbox-gl-draw-rectangle-mode` is safe. If it was alerted by dependabot, we can then directly change the DrawRectangle.js to fix it manually.